sdk/core/core-http/src/util/typeguards.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Helper TypeGuard that checks if something is defined or not.
```

Comment:
I think it's more accurate to say "if the value is not `null` or `undefined`" since one could argue `null` is defined.